lib/jekyll/plugin_manager.rb

```diff
@@ -29,6 +30,15 @@ def require_gems
       )
     end
 
+    def require_theme_deps
+      if site.theme.runtime_dependencies
+        site.theme.runtime_dependencies.each do |dep|
```

Comment:
We tend to prefer guard block style, which is:

```ruby
def require_theme_deps
  return false unless site.theme && site.theme.runtime_dependencies
  # ... do the actual work, expecting the above variables to exist
end
```

You could alternatively use `#Array()`, which will create an empty array if the value is `nil`:

```ruby
def require_theme_deps
  Array(site.theme.runtime_dependencies).each do |dep|
    # ...
  end
end
```